indispensable-cosef/src/commonMain/kotlin/at/asitplus/signum/indispensable/cosef/CoseSigned.kt

```diff
@@ -2,16 +2,24 @@ package at.asitplus.signum.indispensable.cosef
 
```

Comment:
are the changes to this file a complete no-op, or am i missing something?

Reply:
Yes, looks like it, should I rollback any changes?

Reply:
please do, the change set under review should only contain the actual changes. (removing unused imports is fine.)